packages/primitives/README.md

```diff
@@ -0,0 +1,17 @@
+# @headstartwp/blocks-primitives
```

Comment:
@nicholasio I'd love it if we could rename this to `@headstart/block-primitives` to be more aligned with how core names all the packages. :)